test/test_examples.py

```diff
@@ -11,6 +11,7 @@
 EXAMPLES = [
     'baseball.py --num-samples 100 --num-warmup 100',
     'covtype.py --algo hmc --num-samples 10',
+    'hmm.py --num-samples 100 --num-warmup 100',
```

Comment:
I think we'll need to wait for the next version of jax to enable this.

Reply:
Thanks, I add a mark to skip it